### PR TITLE
245 fix ranking in email

### PR DESCRIPTION
**Pull request by kronyer**
I think i did everything :)

### CODE REVIEW

File: TCSA.V2/Components/Admin/Pages/UserDetail.razor [hunk 1]
```diff
@@ -316,10 +316,11 @@
         }
 
         var rankingText = await LeaderboardService.GetUserRanking(UserId);
+        var rankingWithSufix = LeaderboardHelper.GetRanking(rankingText);
```
Comment (stevietv) [inline TCSA.V2/Components/Admin/Pages/UserDetail.razor:319]:
```suggestion
        var rankingWithSuffix = LeaderboardHelper.GetRanking(rankingText);
```

File: TCSA.V2/Components/Admin/Pages/UserDetail.razor [hunk 2]
```diff
@@ -316,10 +316,11 @@
...
 
         var firstName = User.FirstName ??= "";
 
-        var body = EmailHelper.GetProjectCompletedBody(firstName, projectTitle, project.ExperiencePoints, rankingText.ToString());
+        var body = EmailHelper.GetProjectCompletedBody(firstName, projectTitle, project.ExperiencePoints, rankingWithSufix.ToString());
```
Comment (stevietv) [inline TCSA.V2/Components/Admin/Pages/UserDetail.razor:323]:
```suggestion
        var body = EmailHelper.GetProjectCompletedBody(firstName, projectTitle, project.ExperiencePoints, rankingWithSuffix.ToString());
```

File: TCSA.V2/Components/Admin/Pages/UserDetail.razor [hunk 3]
```diff
@@ -334,11 +335,12 @@
 
         var project = ProjectHelper.GetProjects().Single(x => x.Id == 75);
 
-        var rankingText = LeaderboardService.GetUserRanking(UserId);
+        var rankingText = await LeaderboardService.GetUserRanking(UserId);
+        var rankingWithSufix = LeaderboardHelper.GetRanking(rankingText);
```
Comment (stevietv) [inline TCSA.V2/Components/Admin/Pages/UserDetail.razor:339]:
```suggestion
        var rankingWithSuffix = LeaderboardHelper.GetRanking(rankingText);
```

File: TCSA.V2/Components/Admin/Pages/UserDetail.razor [hunk 4]
```diff
@@ -334,11 +335,12 @@
...
 
         var firstName = User.FirstName ??= "";
 
-        var body = EmailHelper.GetProjectCompletedBody(firstName, project.Title, project.ExperiencePoints, rankingText.ToString());
+        var body = EmailHelper.GetProjectCompletedBody(firstName, project.Title, project.ExperiencePoints, rankingWithSufix.ToString());
```
Comment (stevietv) [inline TCSA.V2/Components/Admin/Pages/UserDetail.razor:343]:
```suggestion
        var body = EmailHelper.GetProjectCompletedBody(firstName, project.Title, project.ExperiencePoints, rankingWithSuffix.ToString());
```

**Review by stevietv — CHANGES_REQUESTED** [review]
just a minor typo in the var name

**Comment by kronyer** [comment]
Sorry 😂 not so used to write in english

**Comment by stevietv** [comment]
> Sorry 😂 not so used to write in english

no need to apologize!!!

**Pull request merged**